gallery/src/components/NxStatefulTreeViewRadioSelect/NxStatefulTreeViewRadioSelectPage.tsx

```diff
@@ -22,7 +22,8 @@ const NxStatefulTreeViewRadioSelectPage = () =>
         to <NxCode>NxStatefulCollapsibleRadioSelect</NxCode> and
         is now deprecated and will be removed in the next major version.
         Please use
-        {' '}<NxTextLink href="#/pages/NxStatefulCollapsibleRadioSelect">NxStatefulCollapsibleRadioSelect</NxTextLink>
+        {' '}<NxTextLink href="#/pages/Stateful%20Collapsible%20Radio-Select">NxStatefulCollapsibleRadioSelect
```

Comment:
IMO the cleanest formatting here would be to put the opening tag on its own line, then the content on its own line (indented), and then the closing tag on its own line like it already is